src/apps/gateway/videos/index.test.ts

```
@@ -0,0 +1,193 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { Request, Response } from 'express';
+import { envConfig } from 'src/utils/envConfig';
+import { AppError } from 'src/utils/schema';
+import { validateRequest } from 'src/utils/validator';
+import { logger } from 'src/utils/logger';
+import { createCloudTasks } from '../../../utils/cloud-task';
+import { verifySignature } from '../../../services/videos/convert/validator';
+
+// Mock dependencies
+let routeHandlers: { path: string; middlewares: Function[] }[] = [];
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)